products/mozilla_org/test_mozilla_org.md

```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
 If you work on Pontoon, it is safe to say that it will take less than an hour to see your changes reflected on the dashboard and the dev server.
 
 When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects.
+
+## Testing Dynamic Pages
+This section focuses on instructions for testing pages with dynamically generated content. Each page or topic is different in terms of steps or flow. These instructions could change overtime to reflect product design updates. Linguistic testing is the main focus. The instrutions below are detailed steps to get to the localized content so it can be reviewed in context.
+
+### [firefox/desktop/tips.lang](https://www.mozilla.org/firefox/desktop/tips/)   
+* Click on the **next** or **back** link to check out the tips on different features
+* Highlighted text is the same as in English
+* Text fits inside the box
+
+### [firefox/new/horizon.lang](https://www.mozilla.org/en-US/firefox/new/)
+* Once page is loaded, click on the link called **Download Firefox for another platform**
+* For alternative message, change URL to [Scene2](https://www.mozilla.org/en-US/firefox/new/?scene=2)
+* To fully review the Firefox download page messages, tests must go through following scenarios:
+.. * Desktop: Windows, macOS, Linux
```

Comment:
Use a tab, not .. Checkout out of the page looks like
https://github.com/mozilla-l10n/documentation/blob/933bda12d046a80b2fc255bd0a02ffce7b2d381e/products/mozilla_org/test_mozilla_org.md